bark/tools/dat-from-wav.py

```diff
@@ -0,0 +1,28 @@
+from scipy.io import wavfile as wavfile
```

Comment:
You don't need the `as wavfile` bit here.
`from scipy.io import wavfile` will expose `wavfile` so that you can call `wavfile.read` in the way you want.